.rvmrc

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
not sure if we need this checked in. rvm installations may vary per developer

Reply:
oops. was playing around with gemset. I have removed it